CHANGES

```diff
@@ -1,6 +1,14 @@
 Jinja2 Changelog
 ================
 
+Version 2.9.6
+-------------
+(bugfix release, released on February ???? 2017)
```

Comment:
Considering that it fixes a usecase that was broken in 2.9 it seems somewhat appropriate